### PR TITLE
Prepare Key Vault releases for September 2023.

### PR DESCRIPTION
**Pull request by vcolin7**

### CODE REVIEW

File: eng/jacoco-test-coverage/pom.xml [hunk 1]
```diff
@@ -283,12 +283,12 @@
...
       <groupId>com.azure</groupId>
       <artifactId>azure-security-keyvault-administration</artifactId>
-      <version>4.4.0-beta.1</version> <!-- {x-version-update;com.azure:azure-security-keyvault-administration;current} -->
+      <version>4.4.0</version> <!-- {x-version-update;com.azure:azure-security-keyvault-administration;current} -->
     </dependency>
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-security-keyvault-certificates</artifactId>
-      <version>4.6.0-beta.1</version> <!-- {x-version-update;com.azure:azure-security-keyvault-certificates;current} -->
+      <version>4.5.6</version> <!-- {x-version-update;com.azure:azure-security-keyvault-certificates;current} -->
```
Comment (mssfang) [inline eng/jacoco-test-coverage/pom.xml:291]: should this also be a minor version upgrade? I saw other KV packages are upgraded with a minor version increase.

File: sdk/keyvault/azure-security-keyvault-administration/CHANGELOG.md [hunk 2]
```diff
@@ -1,14 +1,15 @@
...
-### Bugs Fixed
+## 4.4.0 (2023-09-25)
 
 ### Other Changes
+- Explicitly added a `values()` method to all `ExpandableStringEnum` models:
+    - `KeyVaultDataAction`
+    - `KeyVaultRoleDefinitionType`
+    - `KeyVaultRoleScope`
+    - `KeyVaultRoleType`
+- `KeyVaultSettingType`
```
Comment (mssfang) [inline sdk/keyvault/azure-security-keyvault-administration/CHANGELOG.md:11]:
```suggestion
    - `KeyVaultRoleType`
    - `KeyVaultSettingType`
```

File: sdk/keyvault/azure-security-keyvault-administration/CHANGELOG.md [hunk 3]
```diff
@@ -1,14 +1,15 @@
...
-
-### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
+## 4.4.0 (2023-09-25)
 
 ### Other Changes
+- Explicitly added a `values()` method to all `ExpandableStringEnum` models:
```
Comment (mssfang) [inline sdk/keyvault/azure-security-keyvault-administration/CHANGELOG.md:6]: NIT: should we add azure-core and netty upgrade as usual what we had in auto-patch release

File: eng/jacoco-test-coverage/pom.xml (hunk 1 above)
Comment (vcolin7) [reply eng/jacoco-test-coverage/pom.xml:291]: There are no changes that warrant upgrading Certificates to a new minor version in this release.

**Review by mssfang — APPROVED** [review]

**Pull request merged**